nni/compression/pytorch/speedup/v2/model_speedup.py

```diff
@@ -0,0 +1,424 @@
+        #         node_kw = {
+        #             **dict(zip(range(len(arg_list))[1:], node.args)),
+        #             **dict(zip(arg_list[1:], node.args)),
+        #             **{kw_to_posi[k]: v for k, v in node.kwargs.items()},
+        #             **node.kwargs,
+        #         }
+        #         for key, mask in param_masks['_input_'].items():
+        #             node_to_masks[node_kw[key]].append(mask)
+
+        # def check_equal(a, b):
+        #     if type(a) != type(b):
+        #         return False
+        #     if isinstance(a, (list, tuple)):
+        #         if len(a) != len(b):
+        #             return False
+        #         for sub_a, sub_b in zip(a, b):
+        #             if not check_equal(sub_a, sub_b):
+        #                 return False
+        #         return True
+        #     elif isinstance(a, dict):
+        #         if len(set(a.keys()).symmetric_difference(b.keys())) != 0:
+        #             return False
+        #         for key in a.keys():
+        #             if not check_equal(a[key], b[key]):
+        #                 return False
+        #         return True
+        #     else:
+        #         assert isinstance(a, torch.Tensor), f'contents in masks can only be (list, tuple, dict, Tensor), not ({type(a)})'
+        #         return torch.equal(a, b) # totally equal, no bias
+
+        # def check_valid(a, b):
+        #     if isinstance(a, (list, tuple)):
+        #         if not isinstance(b, (list, tuple)):
+        #             return False
+        #         if len(a) != len(b):
+        #             return False
+        #         for sub_a, sub_b in zip(a, b):
+        #             if not check_equal(sub_a, sub_b):
+        #                 return False
+        #         return True
+        #     elif isinstance(a, dict):
+        #         if not isinstance(b, dict):
+        #             return False
+        #         if len(set(a.keys()).symmetric_difference(b.keys())) != 0:
+        #             return False
+        #         for key in a.keys():
+        #             if not check_equal(a[key], b[key]):
+        #                 return False
+        #         return True
+        #     elif isinstance(a, torch.Tensor):
+        #         if not isinstance(b, torch.Tensor):
+        #             return False
+        #         return a.shape == b.shape
+        #     else:
+        #         return b is None
+
+        # for node, masks in node_to_masks.items():
+        #     if len(masks) >= 1:
+        #         mask = masks[0]
+        #         for mask_next in masks[1:]:
+        #             assert check_equal(mask, mask_next), f'preset-masks of "{node}" are not euqal!'
+        #         assert check_valid(self.node_infos[node].output_origin, mask),\
+        #             f'structure of preset-mask and value of slot "{node}" are not euqal!'
+        #         self.node_infos[node].output_masks = mask
```

Comment:
why is this part commented out?

Reply:
pre-set in/out masks is not ready, and I don't want to fix it in this pr